pkg/agent/controller/networkpolicy/cache_test.go

```diff
@@ -619,8 +627,20 @@ func TestRuleCacheAddNetworkPolicy(t *testing.T) {
 			UID:       "policy2",
 		},
 	}
+	networkPolicy3 := &v1beta2.NetworkPolicy{
+		ObjectMeta:      metav1.ObjectMeta{UID: "policy3", Namespace: "ns3", Name: "name3"},
+		Rules:           []v1beta2.NetworkPolicyRule{*networkPolicyRule3},
+		AppliedToGroups: []string{"appliedToGroup1"},
```

Comment:
I still believe that we may be able to omit this field altogether for the rule-level AppliedTo case. I understand why we have it in the controller for span, but it still seems unnecessary to actual send it to the agent.

Reply:
Done.